system/modules/isotope/library/Isotope/Model/Product/Standard.php

```diff
@@ -619,8 +619,8 @@ function (ProductActionInterface $action) use ($arrConfig) {
         $objTemplate->useQuantity = $arrConfig['useQuantity'] && null === $this->getCollectionItem();
         $objTemplate->minimum_quantity = $this->getMinimumQuantity();
         $objTemplate->raw = $this->arrData;
-        $objTemplate->raw_options = $this->getConfiguration();
-        $objTemplate->configuration = $this->getConfiguration();
+        $objTemplate->raw_options = $this->getOptions();
+        $objTemplate->configuration = $this->getOptions();
```

Comment:
why did you change this? Configuration and options is not the same (afair configuration is only for BC)

Reply:
Because it is suggested by PHPDoc 🤷🏻‍♂️

https://github.com/isotope/core/blob/175a07f4b3e84fcfee63d48f8eda91f8475cf512/system/modules/isotope/library/Isotope/Model/Product/Standard.php#L400-L402

Reply:
yes, you should not use the method itself. But they return different values, so we can't replace them in the default template data 🙃

Reply:
@aschempp it's not me, Isotope itself uses it. I get more errors from other places where Isotope calls it BTW. So, deprecated method triggers errors, what do you suggest? 🤔